docs/collections/_schema/schema-grammar.md

```diff
@@ -35,9 +35,11 @@ The grammar ignores whitespace and comments.
 ## `Schema` {#grammar-schema}
 
 A schema consists of a [`NameSpace`](#grammar-schema-NameSpace) JSON object that contains a list of [`EntityTypes`](#grammar-schema-EntityTypes), and a list of [`Actions`](#grammar-schema-Actions).
+The grammar assumes a particular order of keys in JSON objects to simplify the presentation, but this order is not technically required.
+For example, the grammar as written requires that entity type declarations appear before actions, but actions may nonetheless be declared before entity types.
 
 ```
-Schema ::= '{' NameSpace ':' '{' EntityTypes ',' Actions [(',' commonTypes )] '}' '}'
+Schema ::= '{' NameSpace ':' '{' EntityTypes ',' Actions (',' commonTypes )? '}' '}'
```

Comment:
This PR made me realize that we have been inconsistent with our use of `[]` and `?`. We say at the top of https://docs.cedarpolicy.com/schema/schema-grammar.html that `?` is only used for tokens (i.e., STR and IDENT), but then we use it for various grammar constructs (like EntityType). I'm fine with standardizing on `?`, but can you update the text at the top of the file to reflect this?

Reply:
The expression grammar seems to  consistently use `[]`, so it probably make more sense to standardize on that.